.github/workflows/test.yml

```diff
@@ -47,21 +47,10 @@ jobs:
               secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
           echo ::set-output name=vault-token::$VAULT_TOKEN
           echo ::add-mask::$VAULT_TOKEN
-      - name: Get service account credentials from Vault
-        id: vault-secret-step
-        run: |
-          docker run --rm --cap-add IPC_LOCK \
-            -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-            -e "VAULT_ADDR=${VAULT_ADDR}" \
-            vault:1.1.0 \
-            vault read -format json $VAULT_SERVICE_ACCOUNT_PATH \
-            | jq -r .data.key | base64 -d > $SERVICE_ACCOUNT_FILE &&#
-          docker run --rm --cap-add IPC_LOCK \
-            -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-            -e "VAULT_ADDR=${VAULT_ADDR}" \
-            vault:1.1.0 \
-            vault read -format json $VAULT_CLIENT_SERVICE_ACCOUNT_PATH \
-            | jq -r .data.key | base64 -d > $SERVICE_ACCOUNT_CLIENT_FILE
+      - name: Grant execute permission for render-config
+        run: chmod +x render-config.sh
```

Comment:
I think render-config.sh is now also a critical part of local-dev. Can we (probably in a separate PR) add some documentation to the README about using it? should it live in the local-dev directory?

Reply:
I don't feel it is right to move this into local-dev. Because we also need this to run test. 
It is wired that we run local-dev/render-config.sh before running test locally and in github. We should just let the local-dev to run ../render-config.sh first then set the environment variable. WDYT? I will have a seperate PR for that

Reply:
eh, when do we run tests except for "locally." even when the github action runs the test, it runs it "locally" in its container, just from a known clean state. But up to you.

Reply:
okay, moved to local-dev/